data.sql

```diff
@@ -14,4 +14,25 @@ INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
 
 INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Devimon', '05-12-2017', 5, true, 11);
\ No newline at end of file
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '2020-02-08', 0, false, -11);
+
+INSERT INTO animals (name, data_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '2021-09-15', 2, true, -5.7);
+
```

Comment:
- Hey, let's try to pay attention to details as the date is not correct. kindly update your date to `'2021/11/15',`